app/models/job.rb

```diff
@@ -36,7 +36,27 @@ def name
   # The 'slugged' gem expects this method to exist, as it's expecting the
   # method to be an AR attribute.
   def name_changed?
-    title_changed? || company_changed?
+    will_save_change_to_title? || will_save_change_to_company?
+  end
+
+  # Compatibility method for slugged gem due to current Rails changes to the "ActiveModel::Dirty" methods
```

Comment:
I don't know what the "slugged" gem is or why a full on gem was needed for pretty much a `to_param` replacement. All I know is, I hate it.

Anyway, these dynamic methods are no longer in Rails and have replacements. But, the slugged gem hasn't been touched in six years so... yeah.

Reply:
Oh nice!!! Last time I looked into this the "slugged" gem is where I got stuck. like you said, it hadn't been updated in years. I think it's being used to create the slugs in the links for jobs. Love seeing how you got passed that part!

Reply:
Been looking at swapping the slugged gem out for something like [`friendly_id`](https://github.com/norman/friendly_id) here. Might be something to add as a new issue :D